### DISCUSSION TITLE
How to Synchronize Local Git Repository with Latest Remote Branch Names

### DISCUSSION THREAD

**Discussion by dushmanta05** [community/community#68508] [Repositories] [answered]
### Select Topic Area

Question

### Body

Hi! 

I maintain a GitHub repository that currently features the following remote branches:

![remote](https://github.com/community/community/assets/73304098/2232a16b-7d64-41f3-91ca-e72daf1f2fb9)

Previously, the branches `dushmanta-branch` and `pralaya-branch` were named `dushmanta's-branch` and `pralaya's-branch` respectively. I recently corrected these branch names. Additionally, there was a branch named `dushmanta-braanch` which had spelling mistake and subsequently deleted. However, despite these changes, my local repository still retains references to these branches:

![local](https://github.com/community/community/assets/73304098/041442d4-5122-4a59-afd8-2dcb9ddd2d69)

As you can see above, my local repository is still displaying references to branches that have been renamed or deleted. I am seeking help on how to update my local repository to accurately reflect the current state of the remote branches.

I've tried `git fetch` to resolve this issue, but it didn't alter the previously mentioned results.

Any assistance would be greatly appreciated. Thank you.

**Comment by yashSingh97** [comment, marked as answer]
`git remote prune origin` should work

**Reply by dushmanta05** [reply]
@VoidVampire Thank you so much, it worked.
